README.md

```diff
@@ -207,19 +209,18 @@ The AppCredential has a few advantages:
   breach more easy to contain.
 * AppCreds are connected to one project and can be revoked.
 
-We are using an inrestricted AppCred which can create further AppCreds, so
-we can each cluster its own (restricted) credentials. This is not yet
-implemented, but will soon be. Currently, all clusters created from the 
-management node will belong to the same OpenStack project and use the
-same credentials.
+We are using an unrestricted AppCred for the management host which can then create
+further AppCreds, so we can each cluster its own (restricted) credentials.
+In the case of breaches, these AppCreds can be revoked.
 
-The plan for the future is to create AppCreds per cluster 
-(see [#109](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/109)),
-so credentials for individual clusters can be revoked.
+Note that you can have additional projects or clouds listed in your
+``~/.config/openstack/clouds.yaml`` (and ``secure.yaml``) and reference them
+in the ``OPENSTACK_CLOUD`` setting of your ``clusterctl.yaml``, so you can
+manage clusters in various projects and clouds from the same management host.
 
 ## Cluster Management on the C-API management node
 
-You can use ``make ssh`` to log in to the C-API management node. There you can issue
+You can use ``make ssh`` to log in to the CAPI management node. There you can issue
```

Comment:
Consider introducing the `CAPI` abbreviation at the beginning of this README.  
Line 4 could be modified as follows: 
```
[Cluster-API](https://cluster-api.sigs.k8s.io/)(CAPI)
```

IMO it might be worth considering not using the uppercase `CAPI` when we refer to the management node where the CAPI operates. IMO it could be just `capi management node` (note that README contains lowercase `capi` in most cases). CAPI could be used when we reference a Cluster API project itself.

Reply:
Following your suggestion, I use lowercase 'capi management server' as term with lowercase capi and capitalized CAPI when talking about the technology or the project. See [98c48df](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/345/commits/98c48df55d20c7998e66e83b8364b7f23fd65315)
I have no strong opinion on this and would like to get this one merged. Please re-review, @matofeder.

Reply:
Hello @garloff, PR has been re-reviewed and looks good to me.